### PR TITLE
run ci cd on MR

### PR DESCRIPTION
**Pull request by QuangDoo**

### CODE REVIEW

File: .github/workflows/ci-cd.yml
Comment (VanQuang03) [inline .github/workflows/ci-cd.yml:1]: Why do you add this step?

Comment (VanQuang03) [reply .github/workflows/ci-cd.yml:1]: Explain why do you add this step right here? @QuangDoo

Comment (QuangDoo) [reply .github/workflows/ci-cd.yml:1]: I want to run ci-cd on merge request before merging to working branch. @VanQuang03

**Review by VanQuang03 — APPROVED** [review]
approved!

**Pull request merged**